rllib/agents/trainer.py

```diff
@@ -353,6 +349,7 @@ def setup(self, config: PartialTrainerConfigDict):
         # point.
         # Old design: Override `Trainer._init` (or use `build_trainer()`, which
         # will do this for you).
+        _init = True
```

Comment:
this is like super nit picking, apologize if it's too much.
I think it's better to init this to false, and actually set it to True if _init() succeeeds, so.
```
_initialized = False
try:
  self._init()
  _initialized = True
except:
  pass
```
logically feels better.

Reply:
done